packages/next-server/lib/head.js

```diff
@@ -100,4 +100,10 @@ function unique () {
   }
 }
 
+if (process.env.NODE_ENV === 'development') {
```

Comment:
Let's use `!== 'production'` similar to `lib/link.js`

Reply:
```suggestion
if (process.env.NODE_ENV !== 'production') {
```

Reply:
https://github.com/zeit/next.js/blob/canary/packages/next-server/lib/link.js#L155

It looks like `lib/link.js` is using `=== 'development'`. Do you still want me to make the change?

Reply:
Oh interesting, oh well, let's keep it then